apps/system/js/child_window_factory.js

```diff
@@ -143,7 +143,7 @@
     };
     var childWindow = new PopupWindow(configObject);
     childWindow.element.addEventListener('_closing', this);
```

Comment:
can't we just add a
```js
childWindow.element.addEventListener('_opened', this);
```
listener here and benefit from the mechanism we already have in place for activities?
(see `_handle_child__opened` / `_handle_child__closing`)